content/storage/postgres/docker-compose-for-postgres.md

```diff
@@ -38,9 +38,16 @@ services:
 
 # Create name volumes managed by Docker to not lose data when remove a container
 # https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes
+
+# You can also specify a volume driver such as pxd (Portworx) to achieve high performance read/write for container storage volumes.
+https://docs.portworx.com/install-with-other/rancher/rancher-1.x/
 volumes:
   pgdata:
-
+    driver: pxd
+    external: false # If external set to false, the Portworx volume would be automactically created if the volume is not exist.
+    driver_opts:
+       size: 7 # 7 GB
+       repl: 3 # Replicate data across 3 storage volumes
```

Comment:
Enable this option, does it mean we will have 3 volumes that store the same data?

Reply:
Yes, that's right